### ISSUE TITLE
Sudden lag spikes when connect to online servers

### ISSUE THREAD

**Issue by SecretCSGO** [iseekwonderful/csgoGlow#279]
When I use the hack on casual/online servers there are sudden lag spikes. I looked at the other issue posts and saw a few people mention changing the offset or something to turn off the health. I'm very new to this coding stuff. Can someone guide me through turning off the health offset thing? I play CSGO on a mid/late 2014 Macbook Air and am on MacOS Mojave.

**Comment by ethano8225** [comment]
Nope, you shouldn't really need to do that. I have a really old iMac that barely can run CS:GO but it doesn't have any lag spikes that you are talking about. Try using my fork of csglow on my profile (if you use it on MM, the Danger Zone one is here: 
[Danger Zone](https://github.com/jefcostaa/csgoGlow)

**Comment by SecretCSGO** [comment]
ok i will try. Ill let you know anything else if the probem persists. Thanks!

**Comment by SecretCSGO** [comment]
@etan8225 I think im lagging every time i jump or try to bhop. I use a bhop macro btw.

**Comment by sexypoop192** [comment]
@SecretCSGO this is what i did, to stop the lag. press “L-Shift + Tab” close the windows that pop up, close any steam pop ups cuz steam uses a lot of CPU now and when you’re playing, don’t have anything else open. “Command + Q” any other applications that are open. also turn off player health. this can be done by downloading the old template and changing the offsets manually only to show the glow and not player health this can be found in older threads. also don’t use steam voice chat. use in game voice chat. best of luck!

**Comment by SecretCSGO** [comment]
@sexypoop192 can you tell me how to change the offsets? I dont know how

**Comment by SecretCSGO** [comment]
@sexypoop192 can you tell me how to change the offsets? I don't know how. I'm also using Etan's fork of csgoglow.

**Comment by SecretCSGO** [comment]
or which threads can guide me through changing the offsets?

**Comment by SecretCSGO** [comment]
@ghost I've seen some of your comments on other issue threads. Can you help me with changing the offsets so that the health is turned off and only glow?

**Comment by wholivesinapineappleunderthesea** [comment]
@SecretCSGO just change line 77 of main.cpp from: 
`Color color = {float((100 - health) / 100.0), float((health) / 100.0), 0.0f, 0.8f};`
to:
`Color color = {255.0f, 0.0f, 0.0f, 0.8f};`

The structure is r, g, b, alpha. Change it how you please, the example is pure red.

Edit: if you do this the health value is going to be unused and will waste time, so remove line 62 (int health = mem->read<int>(memoryAddress + 0x134);)

**Comment by SecretCSGO** [comment]
@wholivesinapineappleunderthesea Thanks!

**Comment by SecretCSGO** [comment]
wait what about line 73 and 74? xcode says that those 2 lines are "use of undeclared identifier 'health'
do i delete those as well?

**Comment by wholivesinapineappleunderthesea** [comment]
Yes sorry, 73-75 should also be removed
